spring-session/src/main/java/org/springframework/session/data/mongo/MongoHttpSessionConfiguration.java

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
In the other implementations the configuration is in a distinct package to keep configuration distinct from that of the code actually doing stuff. This ensures we do not have any web based dependencies in `org.springframework.session.data.gemfire` (they are all in sub packages). Do you have a good reason for putting this in the same package? If not, it probably makes sense to move this and other configuration related classes into `org.springframework.session.data.mongo.config.annotation.web.http`.